README.md

```diff
@@ -17,19 +17,46 @@
 
 ## API
 ### URL
-> `http://localhost:3001/rooms/:id/` Dynamically render rooms based on listing id.
+> `http://localhost:3001/gallery/:listingId/` Dynamically render gallery based on listing id.
```

Comment:
Would this URL not make more sense as:
```suggestion
> `http://localhost:3001/:listingId/gallery` Dynamically render gallery based on listing id.
```

Reply:
It's fine if it doesn't, I just wanted to offer this as a consideration.

Reply:
Thank you for the suggestion, the page is rendered based on the listing id and not the gallery itself.
 